setup.py

```diff
@@ -4,7 +4,7 @@
 from codecs import open
 from os import path
 
-__version__ = "1.4.1"
+__version__ = "1.4.2"

```

Comment:
Since this has breaking changes I am going to bump `pyetrade` to `2.0.0`. We have two options here either you can bump it using `bumpversion`  or just change this back to `1.4.1` and I will merge it and bump it after. It wont clobber what is currently on PyPI.

Reply:
i reverted the version bump, see latest commit